cmd/siklu/bank_management.c

```diff
@@ -69,4 +73,42 @@ struct software_bank_t* bank_management_get_current_bank(void) {
 	create_bank_management_info_for_bank(bank);
 	
 	return bank;
-}
\ No newline at end of file
+}
+
+int bank_management_get_boot_tries_left() {
+	u_char *fdt = NULL;
+	int8_t boot_tries_left = -1; // negative is disabled
```

Comment:
The function result type is int, but the local variable used for the same purpose is uint8_t.
Is this desired behaviour?

Reply:
Yes, int8_t limits max possible value stored in fdt to a fixed one and platform independent. After adding include types.h we can use int8_t in declaration.

Reply:
fixed